slave.mk

```diff
@@ -30,8 +30,8 @@ DOCKERS_PATH = dockers
 BLDENV = $(shell lsb_release -cs)
 DEBS_PATH = $(TARGET_PATH)/debs/$(BLDENV)
 FILES_PATH = $(TARGET_PATH)/files/$(BLDENV)
-PYTHON_DEBS_PATH = $(TARGET_PATH)/python-debs
-PYTHON_WHEELS_PATH = $(TARGET_PATH)/python-wheels
+PYTHON_DEBS_PATH = $(TARGET_PATH)/python-debs/$(BLDENV)
+PYTHON_WHEELS_PATH = $(TARGET_PATH)/python-wheels/$(BLDENV)
```

Comment:
>PYTHON_WHEELS_PATH [](http://example.com/codeflow?start=0&length=18)

The path changes, and it will break other pipelines if copying artifacts from here. For example: [https://github.com/Azure/sonic-platform-daemons/blob/2b0acfbdc48089acf9224113bbf9a61030314e87/azure-pipelines.yml#L117](https://github.com/Azure/sonic-platform-daemons/blob/2b0acfbdc48089acf9224113bbf9a61030314e87/azure-pipelines.yml#L117)

Do you expect each repo owner to fix the paths?

Reply:
@sujinmkang for awareness.